en/05_Uniform_buffers/00_Descriptor_set_layout_and_buffer.adoc

```diff
@@ -353,15 +326,15 @@ Using a rotation angle of `time * glm::radians(90.0f)` accomplishes the purpose
 
 [,c++]
 ----
-ubo.view = glm::lookAt(glm::vec3(2.0f, 2.0f, 2.0f), glm::vec3(0.0f, 0.0f, 0.0f), glm::vec3(0.0f, 0.0f, 1.0f));
+ubo.view = lookAt(glm::vec3(2.0f, 2.0f, 2.0f), glm::vec3(0.0f, 0.0f, 0.0f), glm::vec3(0.0f, 0.0f, 1.0f));
 ----
 
 For the view transformation I've decided to look at the geometry from above at a 45 degree angle.
 The `glm::lookAt` function takes the eye position, center position and up axis as parameters.
 
 [,c++]
 ----
-ubo.proj = glm::perspective(glm::radians(45.0f), swapChainExtent.width / (float) swapChainExtent.height, 0.1f, 10.0f);
+ubo.proj = glm::perspective(glm::radians(45.0f), static_cast<float>(swapChainExtent.width) / static_cast<float>(swapChainExtent.height), 0.1f, 10.0f);
 ----
 
 I've chosen to use a perspective projection with a 45 degree vertical field-of-view.
```

Comment:
While not part of this PR we should revise this chapter. I don't think manually flipping projection (see line 344+) is required anymore or if it still is there is a GLM define to change that. If this is changed, the whole "orginally for OpenGL" paragraph can be removed.

Reply:
Agree, but let's come back to it after the merge.